baselines/webworker.generated.d.ts

```diff
@@ -2658,15 +2748,23 @@ interface PerformanceObserverCallback {
 declare var onmessage: ((this: DedicatedWorkerGlobalScope, ev: MessageEvent) => any) | null;
 declare function close(): void;
 declare function postMessage(message: any, transfer?: any[]): void;
-declare function dispatchEvent(evt: Event): boolean;
+/**
+     * Dispatches a synthetic event event to target and returns true
+     * if either event's cancelable attribute value is false or its preventDefault() method was not invoked, and false otherwise.
+     */
```

Comment:
This looks strange, hmm.

Reply:
the freaky indentation? that was bothering me too, but couldn't figure out why its this way

Reply:
whoop and `event event`

Reply:
the indentation is because we join the comment lines by an indented `*`, but this comment/statement is top level so shouldn't have any indentation